docs/attributes-registry/file.md

```diff
@@ -0,0 +1,18 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# File
+
+## File Attributes
+
+<!-- semconv registry.file(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `file.directory` | string | Directory where the file is located. It should include the drive letter, when appropriate. | `/home/user`; `C:\Program Files\MyApp` |
```

Comment:
I assume these are coming from https://www.elastic.co/guide/en/ecs/current/ecs-file.html? Just as curiosity, what is the criteria for which fields from ECS are brought over? Or is the intention to bring all of them?

Reply:
Hi @joaopgrassi , yes, these fields are coming from ECS. The intention is to bring them all, but for now I'm concentrating on the fields we need for security rules and use-cases. And later I will continue on adding more fields.